apex/contrib/test/optimizers/test_adam.py

```diff
@@ -0,0 +1,188 @@
+import copy
```

Comment:
could you put this file into tests/L0/run_optimizers?

Reply:
Done